src/app/components/case/decisions/DecisionHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Heading, Spinner } from "@amsterdam/asc-ui"
+import DefinitionList from "app/components/shared/DefinitionList/DefinitionList"
+import { mapArrayToList } from "app/components/shared/Helpers/helpers"
+import { useSummonsWithCaseId } from "app/state/rest/case"
+
+type Props = {
+  caseId: Components.Schemas.Case["id"]
+}
+
+const DecisionHeader: React.FC<Props> = ({ caseId }) => {
+
+  const [data] = useSummonsWithCaseId(caseId)
+  const summon = data?.results?.[0]
+
+  const summonedPersons = summon?.persons
+  const summonTypeName = summon?.type_name
+
+  const mapPersons = summonedPersons?.map((person) => 
+    `${ person.first_name || "" } ${ person.preposition || "" } ${ person.last_name || "" }`
```

Comment:
Please use `??` instead of `||`
And don't we want to show some value like `"Onbekend"` when all three are undefined?

Reply:
Replaced || with ??